lambdas/process_webhooks/constants.py

```diff
@@ -0,0 +1,40 @@
+# Create constants that hold lists of jobs.
```

Comment:
Webhooks from the edx-e2e-test repo are not present here. It's fine to add them in a separate PR, but will they be processed correctly when this goes live?